cmd/agent/common/common_windows.go

```diff
@@ -246,7 +246,10 @@ func ImportRegistryConfig() error {
 	} else {
 		log.Debug("proxy key not found, not setting proxy config")
 	}
-
+	if val, _, err = k.GetStringValue("site"); err == nil {
```

Comment:
Applies to the `DD_URL`-like options as well: if this returns an empty string, we should _not_ set the value on `config.Datadog`. Do we need to add a condition for this here? (`val != ""`)